### ISSUE TITLE
Why abandon ZoeDepth in metric depth fine-tuning of V2?

### ISSUE THREAD

**Issue by liboyu02** [DepthAnything/Depth-Anything-V2#155]
As in title, the code of metric depth estimation in V2 doesn't seem to include ZoeDepth. But in V1, it is obvious to use ZoeDepth. Why this change?

**Comment by LiheYoung** [comment]
Hi @liboyu02, we continue to use ZoeDepth for *generating our metric depth* results in V2, as shown in Table 4. You can reproduce these results using the codebase from V1 by loading the pre-trained weights of V2. The metric depth estimation code in V2 is intended as a simpler alternative to the complex ZoeDepth codebase for training metric depth models. We implemented a basic DPT-based model using a straightforward regression-based loss function for this purpose.

**Comment by I3aer** [comment]
> Hi @liboyu02, we continue to use ZoeDepth for _generating our metric depth_ results in V2, as shown in Table 4. You can reproduce these results using the codebase from V1 by loading the pre-trained weights of V2. The metric depth estimation code in V2 is intended as a simpler alternative to the complex ZoeDepth codebase for training metric depth models. We implemented a basic DPT-based model using a straightforward regression-based loss function for this purpose.

Can you elaborate it further? Do you add another head on DPT or use the available head for relative depth for fine-tuning?
